crmsh/cibconfig.py

```diff
@@ -3158,7 +3158,8 @@ def get_elems_on_ra_type(self, spec):
             tp = content_list[0]
             if not tp:
                 return []
-            return [x for x in self.cib_objects if x.node.get("type") == tp]
+            return [x for x in self.cib_objects if x.node.get("type") == tp or
+                    x.node.get("class") == tp]
```

Comment:
Absolutely better UX! 

Further improvement, I prefer to do partial match to give more flexibility for human-being, something like,

```
tp in x.node.get("class") or tp in x.node.get("provider") or tp in x.node.get("type") 
```

The same for the other two situations: class:provider:type and class:type

Reply:
Also, I would suggest a slightly different title for the git commit log, eg.

```
Dev: cibconfig: "crm config show related:xxx" provides partial search among class, provider, type fields
```

Reply:
@zzhou1 
Thanks! Already changed!